src/amp/components/OnwardContainer.tsx

```diff
@@ -172,6 +167,7 @@ export const OnwardContainer: React.FC<{
                                         <MoustacheSection name="isComment">
                                             <div>
                                                 <MoustacheVariable name="byline" />
+                                                <MoustacheVariable name="byline" />
```

Comment:
all the changes in this file can be undone

Reply:
Good spot @tomrf1 ! I'd forgotten about that!